recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "3.4.2" %}
+{% set md5 = "b180f47bdbebdbbbfe4a6d2d7f95c640" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  entry_points:
+    - coverage = coverage.cmdline:main
+    - coverage3 = coverage.cmdline:main
+    - coverage3.5 = coverage.cmdline:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - coverage
+
+  commands:
+    - coverage --help
+    - coverage3 --help
+    - coverage-3.5 --help
+
+about:
+  home: https://coverage.readthedocs.io
+  license: Apache Software License
+  summary: 'Code coverage measurement for Python'
+
+extra:
+  recipe-maintainers:
+    - ericmjl
```

Comment:
I try not to ping them until we have merged. That way they have the option of participating, but are not obligated to. Also, we have avoided being noisy if possible.